Engine/Log/colors.env

```diff
@@ -112,16 +112,18 @@ puwhite="\033[4;38m"
 pemlightning=" ⚡️ "
 pemfire=" 🔥 "
 pemclouds=" ☁️ "
-pemwarning=" ⚠️ "
```

Comment:
did you checked if this is used anywhere ? 

to avoid any issue we might just duplicates CamelCase en lowercase ?

Reply:
it was colliding with the pemwarning which is lower in the file I just made accessible again